pkg/ddevapp/templates.go

```diff
@@ -18,6 +18,7 @@ services:
         gid: '{{ .GID }}'
     image: ${DDEV_DBIMAGE}-${DDEV_SITENAME}-built
     stop_grace_period: 60s
+    working_dir: "{{ .WebWorkingDir }}"
```

Comment:
Should this be DBWorkingDir?

Reply:
Great catch, thanks. I note that there aren't tests for this, and also found that .DbaWorkingDir needed to be supported.